src/plugins/xerces/README.md

```diff
@@ -2,10 +2,11 @@
 - infos/author = e1528532 <e1528532@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/xml
+- infos/needs =
 - infos/placements = getstorage setstorage
 - infos/status = recommended experimental unittest
+- infos/metadata = elektraXercesOriginalRootName
```

Comment:
Luckily it does not throw away but only does not display it. So I was able to read it in the email, and by pressing `Edit comment` you see it, too.